.mergify.yml

```diff
@@ -1,18 +1,15 @@
 queue_rules:
   - name: default
     conditions:
-      - base=master
-      - status-success=Deck CI
-  - name: backport
-    conditions:
-      - base~=^release-
       - status-success=Deck CI
 
 pull_request_rules:
   - name: Automatically merge on CI success and review
     conditions:
       - 'label=ready to merge'
       - 'approved-reviews-by=@oss-approvers'
+      - base=master
+      - status-success=Deck CI
```

Comment:
since this is already in the `queue_rules`, does it need to be repeated in  each of the `pull_request_rules`?

Reply:
I'm not sure. The example in the blog has it repeated. I asked in the Mergify repo and haven't had a response yet. I guess this would prevent it from bouncing back and forth between the queue and out if the CI is failing?

Reply:
Got a response, apparently it's necessary (*waves hands mysteriously*)